library/cohesity_oracle.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/python
+# Copyright (c) 2018 Cohesity Inc
+# Apache License Version 2.0
+
```

Comment:
can you rename this cohesity_oracle_source, this way we can add cohesity_oracle_job module for protecting this source

Reply:
Done